---
author: Zefeng Zhu
date: 2020-03-07
---

# Structure Resources of SARS-CoV-2

> UPDATE: 2020-03-14

## How many Protein Structures are there?

### Notice in wwPDB

> <http://www.wwpdb.org/news/news?year=2020#5e39e03fa5007a04a313edc3>

A high-resolution crystal structure of COVID-19 (2019-nCoV) coronavirus 3CL hydrolase (Mpro) has been determined by Zihe Rao and Haitao Yang's research team at ShanghaiTech University. Rapid public release of this structure of the main protease of the virus (PDB `6lu7`) will enable research on this newly-recognized human pathogen.

...

While PDB entry `6lu7` is currently the only (__UPDATE: there are 24 now__) public-domain 3D structure from this specific coronavirus, the PDB contains structures of the corresponding enzyme from other coronaviruses. The 2003 outbreak of the closely-related Severe Acute Respiratory Syndrome-related coronavirus (SARS) led to the first 3D structures, and today there are more than 200 PDB structures of SARS proteins. Structural information from these related proteins could be vital in furthering our understanding of coronaviruses and in discovery and development of new treatments and vaccines to contain the current outbreak.

The coronavirus 3CL hydrolase (Mpro) enzyme, also known as the main protease, is essential for proteolytic maturation of the virus. It is thought to be a promising target for discovery of small-molecule drugs that would inhibit cleavage of the viral polyprotein and prevent spread of the infection.

Comparison of the protein sequence of the COVID-19 coronavirus 3CL hydrolase (Mpro) against the PDB archive identified 95 PDB proteins with at least __90%__ sequence identity. Furthermore, these related protein structures contain approximately 30 distinct small molecule inhibitors, which could guide discovery of new drugs. Of particular significance for drug discovery is the very high amino acid sequence identity (96%) between the COVID-19 coronavirus 3CL hydrolase (Mpro) and the SARS virus main protease (PDB 1q2w). __Summary data about these closely-related PDB structures are available (PDB_proteins_90pc_identity_6lu7.csv)__ to help researchers more easily find this information. In addition, the PDB houses 3D structure data for more than 20 unique SARS proteins represented in more than 200 PDB structures, including a second viral protease, the RNA polymerase, the viral spike protein, a viral RNA, and other proteins __(PDB_entries_SARS_grouped.csv)__.

#### Update: Released COVID-19-related PDB structures include

* PDB structure `6lu7` 
  * (X. Liu, B. Zhang, Z. Jin, H. Yang, Z. Rao Crystal structure of COVID-19 main protease in complex with an inhibitor N3 doi: 10.2210/pdb6lu7/pdb) 
  * Released 2020-02-05
* PDB structure `6vsb` 
  * (D. Wrapp, N. Wang, K.S. Corbett, J.A. Goldsmith, C.-L. Hsieh, O. Abiona, B.S. Graham, J.S. McLellan (2020) Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation Science doi: 10.1126/science.abb2507) 
  * Released 2020-02-26
* PDB structure `6lxt` 
  * (Y. Zhu, F. Sun Structure of post fusion core of 2019-nCoV S2 subunit doi: 10.2210/pdb6lxt/pdb) 
  * Released 2020-02-26
* PDB structure `6lvn` 
  * (Y. Zhu, F. Sun Structure of the 2019-nCoV HR2 Domain doi: 10.2210/pdb6lvn/pdb) 
  * Released 2020-02-26
* PDB structure `6vw1`
  * J. Shang, G. Ye, K. Shi, Y.S. Wan, H. Aihara, F. Li Structural basis for receptor recognition by the novel coronavirus from Wuhan doi: 10.2210/pdb6vw1/pdb
  * Released 2020-03-04
* PDB structure `6vww`
  * Y. Kim, R. Jedrzejczak, N. Maltseva, M. Endres, A. Godzik, K. Michalska, A. Joachimiak, Center for Structural Genomics of Infectious Diseases Crystal Structure of NSP15 Endoribonuclease from SARS CoV-2 doi: 10.2210/pdb6vww/pdb
  * Released 2020-03-04
* PDB structure `6y2e`
  * L. Zhang, X. Sun, R. Hilgenfeld Crystal structure of the free enzyme of the SARS-CoV-2 (2019-nCoV) main protease doi: 10.2210/pdb6y2e/pdb
  * Released 2020-03-04
* PDB structure `6y2f`
  * L. Zhang, X. Sun, R. Hilgenfeld Crystal structure (monoclinic form) of the complex resulting from the reaction between SARS-CoV-2 (2019-nCoV) main protease and tert-butyl (1-((S)-1-(((S)-4-(benzylamino)-3,4-dioxo-1-((S)-2-oxopyrrolidin-3-yl)butan-2-yl)amino)-3-cyclopropyl-1-oxopropan-2-yl)-2-oxo-1,2-dihydropyridin-3-yl)carbamate (alpha-ketoamide 13b) doi: 10.2210/pdb6y2f/pdb
  * Released 2020-03-04
* PDB structure `6y2g`
  * L. Zhang, X. Sun, R. Hilgenfeld Crystal structure (orthorhombic form) of the complex resulting from the reaction between SARS-CoV-2 (2019-nCoV) main protease and tert-butyl (1-((S)-1-(((S)-4-(benzylamino)-3,4-dioxo-1-((S)-2-oxopyrrolidin-3-yl)butan-2-yl)amino)-3-cyclopropyl-1-oxopropan-2-yl)-2-oxo-1,2-dihydropyridin-3-yl)carbamate (alpha-ketoamide 13b) doi: 10.2210/pdb6y2g/pdb
  * Released 2020-03-04
* ...

### Notice in RCSB

> <http://www.rcsb.org/news?year=2020&article=5e3c4bcba5007a04a313edcc>

> PDB Structures: [Access all 24 COVID19-related PDB structures](http://www.rcsb.org/pdb/results/results.do?tabtoshow=Current&qrid=8045C48A)

<table>
    <tr>
        <td>
            The <b>asymmetric unit</b> of PDB ID <a href="http://www.rcsb.org/pdb/explore.do?structureId=6lu7">6lu7</a>
        </td>
        <td>
            The <b>biological assembly</b> of PDB ID <a href="http://www.rcsb.org/pdb/explore.do?structureId=6lu7">6lu7</a>
        </td>
    </tr>
    <tr>
        <td>
            <img src="https://cdn.rcsb.org/images/rutgers/lu/6lu7/6lu7.pdb-500.jpg"/>
        </td>
        <td>
            <img src="https://cdn.rcsb.org/images/rutgers/lu/6lu7/6lu7.pdb1-500.jpg"/>
        </td>
    </tr>
</table>

### Notice in SWISS-MODEL

> <https://swissmodel.expasy.org/repository/species/2697049>

> We modelled the full SARS-CoV-2 proteome based on the NCBI reference sequence NC_045512 which is identical to GenBank entry MN908947, and annotations from UniProt. The underlying SWISS-MODEL workspace projects are accessible below and, as a collaborative effort, new models exploring alternative templates or conformational states can be built by anyone. Our colleagues at the Swiss Institute of Bioinformatics provide more resources on the SARS-CoV-2 genome on ViralZone or allow to track the current outbreak on Nextstrain. Several SARS-CoV-2 proteins reportedly interact in hetero-oligomeric complexes for which we also provide models in a dedicated hetero section. But be aware: the result of any theoretical modelling procedure is NON-EXPERIMENTAL and must be considered with care as models may contain significant errors. To this end SWISS-MODEL provides quality metrics which are described in the help section. The provided modelling projects are based on SWISS-MODEL template library version 2020-03-11 and will be updated as new templates become available.

### Notice in Rosetta

> <https://www.ipd.uw.edu/2020/02/rosettas-role-in-fighting-coronavirus/>

> <https://www.ipd.uw.edu/coronavirus/>

> <http://new.robetta.org/results.php?id=15652>

### Notice in Yang Zhang's Research Group

> <https://zhanglab.ccmb.med.umich.edu/C-I-TASSER/2019-nCov/>

#### Structure models of all mature peptides in 2019-nCoV genome by C-I-TASSER

> This page contains 3D structural models and function annotation for all proteins encoded by the genome of 2019-nCoV, also known as SARS-CoV-2, which is the novel coronavirus causing the COVID-19 pneumonia. The structure models are generated by the C-I-TASSER pipeline, which utilizes deep convolutional neural-network based contact-map predictions to guide the I-TASSER fragment assembly simulations. Benchmark and blind CASP tests showed that C-I-TASSER generates models with a higher accuracy than I-TASSER does, especially for the protein targets lack of homologous templates. For multi-domain targets, the C-I-TASSER structure of individual domains are assembled by DEMO into full length structure.

#### Updates

* 2020/02/16: COFACTOR predicted GO terms and EC numbers are available [here](https://zhanglab.ccmb.med.umich.edu/C-I-TASSER/2019-nCov/COFACTOR). COACH predicted ligand binding sites are available [here](https://zhanglab.ccmb.med.umich.edu/C-I-TASSER/2019-nCov/COACH).
* 2020/02/12: Spike complex structures are uploaded.
* 2020/02/08: A preprint for computational analysis of spike-ACE2 interaction and intermediate host identification is posted here.
* 2020/02/01: Domain assembly of QHD43415_11 is found to be incorrect. A new model is uploaded to replace the old model.
* 2020/01/31: Initial release.

#### Reference

> Chengxin Zhang, Wei Zheng, Xiaoqiang Huang, Eric W. Bell, Xiaogen Zhou, Yang Zhang (2020) "Protein structure and sequence re-analysis of 2019-nCoV genome does not indicate snakes as its intermediate host or the unique similarity between its spike protein insertions and HIV-1." doi: http://dx.doi.org/10.1101/2020.02.04.933135

### Notice in DeepMind

> https://deepmind.com/research/open-source/computational-predictions-of-protein-structures-associated-with-COVID-19

#### Reference

> John Jumper, Kathryn Tunyasuvunakool, Pushmeet Kohli, Demis Hassabis, and the AlphaFold Team, “Computational predictions of protein structures associated with COVID-19”, DeepMind website, 5 March 2020, https://deepmind.com/research/open-source/computational-predictions-of-protein-structures-associated-with-COVID-19